From 44220b8d7cd2146a613c3d0bd772d538a455f56a Mon Sep 17 00:00:00 2001
From: "Tekriwal, Tanuj" <tanuj.tekriwal@intel.com>
Date: Fri, 11 Apr 2025 11:17:22 +0000
Subject: [PATCH] Update the clang version to latest on A15

This patch will update the clang version.

Tests Done: Build and boot

Tracked-On: NA
Signed-off-by: Tekriwal, Tanuj <tanuj.tekriwal@intel.com>
---
 groups/kernel/AndroidBoard.mk                                   | 2 +-
 .../gmin64/config-lts/linux-intel-lts2023/x86_64_defconfig      | 2 +-
 .../kernel/gmin64/config-lts/lts2023-chromium/x86_64_defconfig  | 2 +-
 .../kernel/gmin64/config-lts/lts2024-android/x86_64_defconfig   | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/groups/kernel/AndroidBoard.mk b/groups/kernel/AndroidBoard.mk
index 0802739..f9e4e87 100644
--- a/groups/kernel/AndroidBoard.mk
+++ b/groups/kernel/AndroidBoard.mk
@@ -6,7 +6,7 @@ PREBUILT_KERNEL_ROOT := vendor/intel/utils_priv/kernel/prebuilts/6.6/{{{prebuilt
 endif
 endif
 
-TARGET_KERNEL_CLANG_VERSION := r530567
+TARGET_KERNEL_CLANG_VERSION := r530567b
 CLANG_PREBUILTS_PATH := $(abspath $(INTEL_PATH_DEVICE)/../../../prebuilts/clang)
 
 ifneq ($(TARGET_KERNEL_CLANG_VERSION),)
diff --git a/groups/kernel/gmin64/config-lts/linux-intel-lts2023/x86_64_defconfig b/groups/kernel/gmin64/config-lts/linux-intel-lts2023/x86_64_defconfig
index 7befe4d..95014e6 100644
--- a/groups/kernel/gmin64/config-lts/linux-intel-lts2023/x86_64_defconfig
+++ b/groups/kernel/gmin64/config-lts/linux-intel-lts2023/x86_64_defconfig
@@ -2,7 +2,7 @@
 # Automatically generated file; DO NOT EDIT.
 # Linux/x86_64 6.6.68 Kernel Configuration
 #
-CONFIG_CC_VERSION_TEXT="Android (12328485, +pgo, +bolt, +lto, +mlgo, based on r530567) clang version 19.0.0 (https://android.googlesource.com/toolchain/llvm-project 97a699bf4812a18fb657c2779f5296a4ab2694d2)"
+CONFIG_CC_VERSION_TEXT="Android (12642944, +pgo, +bolt, +lto, +mlgo, based on r530567b) clang version 19.0.0 (https://android.googlesource.com/toolchain/llvm-project 97a699bf4812a18fb657c2779f5296a4ab2694d2)"
 CONFIG_GCC_VERSION=0
 CONFIG_CC_IS_CLANG=y
 CONFIG_CLANG_VERSION=190000
diff --git a/groups/kernel/gmin64/config-lts/lts2023-chromium/x86_64_defconfig b/groups/kernel/gmin64/config-lts/lts2023-chromium/x86_64_defconfig
index 6beb66d..9aab9c4 100644
--- a/groups/kernel/gmin64/config-lts/lts2023-chromium/x86_64_defconfig
+++ b/groups/kernel/gmin64/config-lts/lts2023-chromium/x86_64_defconfig
@@ -2,7 +2,7 @@
 # Automatically generated file; DO NOT EDIT.
 # Linux/x86_64 6.6.56 Kernel Configuration
 #
-CONFIG_CC_VERSION_TEXT="Android (12328485, +pgo, +bolt, +lto, +mlgo, based on r530567) clang version 19.0.0 (https://android.googlesource.com/toolchain/llvm-project 97a699bf4812a18fb657c2779f5296a4ab2694d2)"
+CONFIG_CC_VERSION_TEXT="Android (12642944, +pgo, +bolt, +lto, +mlgo, based on r530567b) clang version 19.0.0 (https://android.googlesource.com/toolchain/llvm-project 97a699bf4812a18fb657c2779f5296a4ab2694d2)"
 CONFIG_GCC_VERSION=0
 CONFIG_CC_IS_CLANG=y
 CONFIG_CLANG_VERSION=190000
diff --git a/groups/kernel/gmin64/config-lts/lts2024-android/x86_64_defconfig b/groups/kernel/gmin64/config-lts/lts2024-android/x86_64_defconfig
index df2595a..083b476 100644
--- a/groups/kernel/gmin64/config-lts/lts2024-android/x86_64_defconfig
+++ b/groups/kernel/gmin64/config-lts/lts2024-android/x86_64_defconfig
@@ -2,7 +2,7 @@
 # Automatically generated file; DO NOT EDIT.
 # Linux/x86_64 6.12.8 Kernel Configuration
 #
-CONFIG_CC_VERSION_TEXT="Android (12328485, +pgo, +bolt, +lto, +mlgo, based on r530567) clang version 19.0.0 (https://android.googlesource.com/toolchain/llvm-project 97a699bf4812a18fb657c2779f5296a4ab2694d2)"
+CONFIG_CC_VERSION_TEXT="Android (12642944, +pgo, +bolt, +lto, +mlgo, based on r530567b) clang version 19.0.0 (https://android.googlesource.com/toolchain/llvm-project 97a699bf4812a18fb657c2779f5296a4ab2694d2)"
 CONFIG_GCC_VERSION=0
 CONFIG_CC_IS_CLANG=y
 CONFIG_CLANG_VERSION=190000
-- 
2.34.1

